crates/nu-cli/src/shell/palette.rs

```diff
@@ -89,7 +91,7 @@ impl Palette for ThemedPallet {
             FlatShape::Type => single_style_span(self.theme.r#type.bold(), shape.span),
             FlatShape::Operator => single_style_span(self.theme.operator.normal(), shape.span),
             FlatShape::DotDot => single_style_span(self.theme.dot_dot.bold(), shape.span),
-            FlatShape::Dot => single_style_span(Style::new().fg(self.theme.dot), shape.span),
+            FlatShape::Dot => single_style_span(Style::new().fg(*self.theme.dot), shape.span),
```

Comment:
Is there a way to avoid the explicit dereference here?

Reply:
You could use `self.theme.dot.normal()` I think